style2.scss

```diff
@@ -0,0 +1,367 @@
+$color-dark: #000;
+$color-white: #fff;
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  background-repeat: no-repeat;
+  box-sizing: inherit;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+// Navigation
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 1rem;
+  background-color: $color-dark;
+  color: $color-white;
+
+  p {
+    font-size: 1.4rem;
+    margin: 0.8rem 0;
+  }
+
+  ul {
+    list-style-type: none;
+
+    li {
+      font-size: 0.8rem;
+      display: inline;
+      text-transform: uppercase;
+      padding-left: 1rem;
+
+      a {
+        text-decoration: none;
+        color: $color-white;
+      }
+    }
+  }
+}
+
+// Header
+.header {
+  height: 90vh;
+  .header-background {
+    z-index: 1;
+    height: 100%;
+    width: 100%;
+    background-position: center;
+    background-size: contain;
+    margin-top: 0.5rem;
+    /* blur */
+    filter: blur(2px);
+    -webkit-filter: blur(2px);
+  }
+  .header-text {
+    font-size: 1.5rem;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 2;
+    width: 80%;
+    padding: 1.25rem;
+    text-align: center;
+  }
+}
+
+// About
+.about {
+  height: 110vh;
+  margin: 0rem 2rem;
+
+  .about-left {
```

Comment:
This is SCSS. You can do something like this 
`.about {
      height: 110vh;
      margin: 0rem 2rem;

    &-left {...}
    
    &-right {...}
}
`